sklearn/neighbors/_ball_tree.pyx

```diff
@@ -14,7 +14,8 @@ VALID_METRICS = ['EuclideanDistance', 'SEuclideanDistance',
                  'DiceDistance', 'KulsinskiDistance',
                  'RogersTanimotoDistance', 'RussellRaoDistance',
                  'SokalMichenerDistance', 'SokalSneathDistance',
-                 'PyFuncDistance', 'HaversineDistance']
+                 'ArccosDistance', 'HaversineDistance',
```

Comment:
To match the order of `DistanceMetric`'s definition in `sklearn/metrics/_dist_metrics.pyx.tp`
```suggestion
                 'HaversineDistance', 'ArccosDistance',
```